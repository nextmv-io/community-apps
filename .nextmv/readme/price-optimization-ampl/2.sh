cat input.json | docker run -i --rm \
-v $(pwd):/app ghcr.io/nextmv-io/runtime/ampl:latest \
python3 /app/main.py
