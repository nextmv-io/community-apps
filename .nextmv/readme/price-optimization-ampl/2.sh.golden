{
  "solutions": [
    {
      "regions": [
        "great_lakes",
        "midsouth",
        "northeast",
        "northern_new_england",
        "southcentral",
        "southeast",
        "west",
        "plains"
      ],
      "price": {
        "great_lakes": 1.78,
        "midsouth": 1.51,
        "northeast": 2.0,
        "northern_new_england": 1.59,
        "southcentral": 2.0,
        "southeast": 1.9,
        "west": 2.0,
        "plains": 1.21
      },
      "quantity": {
        "great_lakes": 3.50819768,
        "midsouth": 3.94327922,
        "northeast": 4.46022391,
        "northern_new_england": 0.917984,
        "southcentral": 4.74101614,
        "southeast": 3.84072787,
        "west": 5.62900352,
        "plains": 2.95956766
      }
    }
  ],
  "statistics": {
    "result": {
      "custom": {
        "constraints": 41,
        "provider": "highs",
        "status": "optimal",
        "variables": 56,
        "expected_demand": {
          "great_lakes": 3.50819768,
          "midsouth": 3.8645434,
          "northeast": 4.46022391,
          "northern_new_england": 0.917984,
          "southcentral": 6.48841261,
          "southeast": 3.61119424,
          "west": 5.85853716,
          "plains": 1.2121712
        },
        "expected_sales": {
          "great_lakes": 3.50819768,
          "midsouth": 3.8645434,
          "northeast": 4.46022391,
          "northern_new_england": 0.917984,
          "southcentral": 4.74101614,
          "southeast": 3.84072787,
          "west": 5.62900352,
          "plains": 2.95956766
        },
        "expected_waste": {
          "great_lakes": 0.0,
          "midsouth": 0.07873582,
          "northeast": 0.0,
          "northern_new_england": 0.0,
          "southcentral": 0.0,
          "southeast": 0.0,
          "west": 0.0,
          "plains": 0.0
        }
      },
      "duration": 0.123,
      "value": 46.4511927332535
    },
    "run": {
      "duration": 0.123,
      "custom": {
        "license_used": "demo"
      }
    },
    "schema": "v1"
  }
}
